requirements.txt

```diff
@@ -12,6 +12,7 @@
     certifi~=2020.12.5
     multidict~=5.0.0
     numpy~=1.23
+    notifypy>=1.0.3.0,
```

Comment:
I suspect the comma on this line to cause problems

Reply:
i think notifypy does install notify-py under the hood?